packages/video_player/lib/video_player.dart

```diff
@@ -419,6 +432,33 @@ class VideoPlayerController extends ValueNotifier<VideoPlayerValue> {
     value = value.copyWith(volume: volume.clamp(0.0, 1.0));
     await _applyVolume();
   }
+
+  Future<void> _applySpeed() async {
+    if (!value.initialized || _isDisposed) {
+      return;
+    }
+
+    // On iOS setting the speed will start playing the video automatically
+    // Do not change the video speed until after the video is played
+    if (!value.isPlaying) {
+      return;
+    }
+
+    // ignore: strong_mode_implicit_dynamic_method
+    await _channel.invokeMethod(
```

Comment:
```suggestion
    await _channel.invokeMethod<void>(
```

You can remove the comment above if you change to this